data/h3_raster_importer/tiff_folder_to_h3_table.py

```diff
@@ -89,6 +89,7 @@ def gen_raster_h3(raster_list, h3_res, geo=False):
                     raise ValueError("Transforms do not match")
                 arr = src.read(1, window=window)
                 _df = h3ronpy.raster.raster_to_dataframe(arr, w_transform, h3_res, nodata_value=src.profile['nodata'], compacted=False, geo=geo)
+                _df = _df.fillna(-9999)
```

Comment:
Maybe this value should be made a script parameter?

Reply:
Also why are you filling the nodata back with -9999? Couldn't these be left to go into the DB as `null`? (or maybe even zero would be correct for the crop data)

Reply:
I'm setting the no data to -9999  for the same reason explained below - I did the same in this part of the code because (I don't know the reason why) I end up with NA values from the raster to h3 transformation even though I have already set the values to -9999 in the main script. I'll have another look today.

I didn't set this value to 0 to differentiate the no data from the areas with actual 0 value. I can set it to null or -1 as we have in mapspam.

Reply:
Hmm, ok I think it would be best to save them as nodata in the DB, but if they're not importing to the DB, maybe we need to configure the csv export like this: https://stackoverflow.com/questions/50890989/pandas-changing-the-format-of-nan-values-when-saving-to-csv?

Reply:
let me have a look

Reply:
ok, changed the sql for importing the data to accept no data as 'NULL'.